docs/detections/rules-ui-create.asciidoc

```diff
@@ -266,9 +266,13 @@ NOTE: To preview rules, you need the `read` privilege to the `.preview.alerts-se
 To preview a rule:
 
 . Write the rule query.
+. Choose how you want to preview the query results:
+
+** *Quick query preview*: Allows you to select from pre-defined timeframes -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. Note that threshold and event correlation rules have limited timeframe options. The rule interval and look-back time are also pre-defined for the preview and differ by rule type. These settings cannot be modified.
+** *Advanced query preview*: Allows you to choose a custom timeframe for the rule preview, schedule how often the rule should run, and specify a look-back time.
```

Comment:
Minor edit suggestion to reduce repetition and filler. Also, looks like two-word "time frame" is more prevalent in our docs.
```suggestion
** *Quick query preview*: Select from pre-defined time frames -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. Note that threshold and event correlation rules have limited time frame options. The rule interval and look-back time are also pre-defined for the preview and differ by rule type. These settings cannot be modified.
** *Advanced query preview*: Choose a custom time frame for the rule preview, schedule how often the rule should run, and specify a look-back time.
```